test/change_streams/unified/change-streams-errors.json

```diff
@@ -187,7 +187,7 @@
       "description": "change stream errors on ElectionInProgress",
       "runOnRequirements": [
         {
-          "minServerVersion": "4.4",
+          "minServerVersion": "4.2",
```

Comment:
We're pinning this to 4.4 pending https://jira.mongodb.org/projects/PYTHON/issues/PYTHON-3292